chokuretsu/guide/dump-rom.html

```diff
@@ -0,0 +1,294 @@
+                <li>Launch Twilight Menu++ as you were shown and run GodMode9i.</li>
+                <li>Select the "NDS GAMECARD" option.</li>
+                <li>Press the <i class="fa-brands fa-adn"></i> Button to select "yes" to dump your cartridge.</li>
+                <li>When complete, you can turn off your DSi and remove the SD card from the system.</li>
+                <li>Insert the SD card back into your computer. Navigate to <code>/gm9i/out/</code> on your SD card.
+                    Copy the <code>.nds</code> file to your computer.
+                </li>
+            </ol>
+        </div>
+
+        <!-- Nintendo 3DS dump -->
+        <div class="one-column-text hidden 3ds-show emu-show">
+            <h3 id="dump-3ds">Dumping with a Nintendo 3DS</h3>
+            <b>Difficulty:</b> <i class="fa-solid fa-star"></i><i class="fa-regular fa-star"></i><i
+                class="fa-regular fa-star"></i><br/>
+            <p>To dump using a Nintendo 3DS or 2DS console, we will need to install a Custom Firmware (CFW) onto the
+                system known as Luma3DS and bootstrap9 to let us run Godmode9, which lets us dump the cartridge
+                to an <code>.nds</code> file.</p>
+
+            <b>Requirements:</b>
+            <ul>
+                <li>Nintendo 3DS, 3DS XL, 2DS, New 3DS, New 3DS XL or New 2DS XL console</li>
+                <li><a href="buy-game.html"><i>Suzumiya Haruhi no Chokuretsu</i></a></li>
+                <li>SD card with at least 2GB capacity</li>
+                <li>Computer with SD card reader/USB SD card reader peripheral</li>
+                <li>In some cases, you may need to use a DS or DSi-compatible flash cartridge.</li>
+            </ul>
+            <b>Method:</b>
+            <ol>
+                <li>Follow the steps <a href="https://3ds.hacks.guide/get-started" target="_blank"
+                                        class="exclude-device-href">
+                    on 3ds.hacks.guide</a> to install Luma3DS and bootstrap9 onto your 3DS.
+                </li>
+                <li>Continue <a href="https://3ds.hacks.guide/finalizing-setup" target="_blank"
+                                class="exclude-device-href">following the guide to
+                    install GodMode9</a> as well as other prerequisite homebrew software onto your 3DS.
+                </li>
+                <li>Power off your console (not just sleep mode, a full power-off)</li>
+                <li>Insert the <i>Suzumiya Haruhi no Chokuretsu</i> game cartridge into your 3DS.</li>
+                <li>Hold the START button and press the <i class="fa-solid fa-power-off"></i> Button to power on the
+                    console. This should launch GodMode9. If it doesn't, power off and try again.
+                </li>
+                <li>Use the D-Pad to navigate to <code>[C:] GAMECART</code></li>
+                <li>Press the <i class="fa-brands fa-adn"></i> Button on <code>[TitleID].nds</code> to select it. When
```

Comment:
```suggestion
                <li>Press the <i class="fa-brands fa-adn"></i> Button on <code>[TitleID].nds</code> to select it. If
```

Had my girlfriend try this out -- this step confused her as she wasn't prompted to select a dump type (possibly bc I already had selected it in previous sessions?)